v1.1/parallel-statement-execution.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Parallel Statement Execution
+summary: The parallel statement execution feature allows parallel execution of multiple independent SQL statements within a transaction.
+toc: false
+---
+
+CockroachDB supports parallel execution of independent INSERT, UPDATE, UPSERT, and DELETE statements within a single transaction. 
+
+To execute statements in parallel, use the `RETURNING NOTHING` clause for the independent SQL statements.
```

Comment:
We changed it from 'RETURNING NOTHING parameter' to 'RETURNING NOTHING clause' just yesterday :) Need to decide on the terminology. I would still prefer 'parameter' or 'clause' because that's what we use in other docs.